ReactWindows/ReactNative/Views/WebView/ReactWebViewManager.cs

```diff
@@ -0,0 +1,289 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Controls;
+using ReactNative.Views.WebView.Events;
+using Windows.Web.Http;
+
+namespace ReactNative.Views.WebView
+{
+    /// <summary>
+    /// A view manager responsible for rendering webview.
+    /// </summary>
+    public class ReactWebViewManager : SimpleViewManager<Windows.UI.Xaml.Controls.WebView>
+    {
+        private const string BLANK_URL = "about:blank";
+
+        private const int CommandGoBack = 1;
+        private const int CommandGoForward = 2;
+        private const int CommandReload = 3;
+
+        private Dictionary<int, string> _injectedJS = new Dictionary<int, string>();
```

Comment:
Make sure this is readonly, I'm not a huge fan of keeping this dictionary here, but unfortunately with WebView being sealed, we don't have many other options.